packages/@tinacms/toolkit/src/packages/fields/components/ImageUpload/ImageUpload.tsx

```diff
@@ -147,9 +147,9 @@ export const ImageUpload = ({
             <ImageLoadingIndicator />
           ) : (
             <ImgPlaceholder>
-            Drag 'n' drop some files here,
+            Drag 'n' drop a file here,
             <br />
-            or click to select files
+            or click to select a file
           </ImgPlaceholder>
```

Comment:
Thinking about the image field, it really should only accept one file in this instance. I just updated the language to avoid confusion.